v2/third_party/uw-labs/lichen/buildinfo/parse.go

```diff
@@ -0,0 +1,93 @@
+package buildinfo
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/google/go-licenses/v2/third_party/uw-labs/lichen/model"
+)
+
+// Parse parses build info details as returned by `go version -m [bin ...]`
+func Parse(info string) ([]model.BuildInfo, error) {
```

Comment:
Lifted this idea from https://github.com/golang/go/issues/39301 - but Go has an equivalent impl in https://cs.opensource.google/go/go/+/refs/tags/go1.16.6:src/runtime/debug/mod.go;l=37. We'd have to make some modifications, but this would be a trimmed down library direct from the Go team.

Since this is likely going to be the impl that https://github.com/golang/go/issues/39301 is based off of, thoughts on using this over lichen?

Reply:
Fair enough, I'll use this version of code instead.

Reply:
Done